### ISSUE TITLE
 overvaluation of "bait"

### ISSUE THREAD

**Issue by keybounce** [Grimy/Grimy.github.io#34]
so I finally figured out why bait is being overvalued.

			// number of ticks needed to repopulate an army
			let timer = log(1 + soldiers() * breed() / add(Bait, 100)) / log(1 + breed());

The truth is, trapping doesn't have any value later in the game. The speed that you can open a trap maxes out at once per tick. So the base breeding speed completely overwhelms the bait/trapping speed.

My HCE is 60. I spent most of my time playing decay at the moment. Even before I had decay unlocked, I was still noticing this whole thing that auto traps were useless, because using trapping to try to repopulate my army did not have a significant effect (less than one second, even with yellow 12 repopulation time).

In other words, bait/trapping has value in the early game, when 10 or 20 starting traps are being used to populate my initial army, or when my initial flood of mansions causes a large need of more trimps. But not late game, which seems to be what perky is based on.

**Comment by Grimy** [comment]
Why do you quote that line of code? Do you believe it incorrectly computes the time needed to repopulate an army?

You’re correct that trapping doesn’t have any significant effect past the very early game (z40 or so). Perky also gets to the same conclusion, which is why it recommends very little Bait if you set your target zone high enough.

**Comment by keybounce** [comment]
I quote that line of code because it looks like it is describing the amount of time that it takes to repopulate the Army, and because it is using the bait value as part of the calculation.

It is dividing by the value of bait, without taking any limit of reproduction into account.

In other words, the effectiveness of bait is not determined by "add (bait, 100)", but by a smaller number that represents the behavior that your trapping speed hits a limit even as player effectiveness should continue to increaseJ Is okay

**Comment by Grimy** [comment]
The player can’t open more than 10 traps per second regardless of efficiency, and this cap is reached fairly quickly. Perky assumes that you’re constantly at the cap, and so the overall trapping speed is `10 * (Bait level + 1)` trimps per second. Given this assumption, the calculation is correct.
